site/_posts/2021-03-16-securely-implement-oauth-vuejs.md

```diff
@@ -349,7 +380,7 @@ Create a new file called `Greeting.vue` in the `src` folder.  We will add logic
 ```vue
 <template>
   <div class="greet">
-    <h3 v-if="email">Welcome {{email}} </h3>
+    <h3 v-if="email">Welcome{% raw %} {{email}} {% endraw %}</h3>
```

Comment:
This is a jekyll thing (`{%raw%}`) correct?

Reply:
Seems to update correctly... the previous version was not allowing `{{ email }}` to render in the code block.  attached is how it renders on the site.  

<img width="926" alt="Screen Shot 2021-03-22 at 2 09 54 PM" src="https://user-images.githubusercontent.com/16090626/112052110-5961f500-8b18-11eb-93c2-0c13be049e8f.png">

It seems to copy ok: 
```
<template>
  <div class="greet">
    <h3 v-if="email">Welcome {{email}} </h3>
    <h3 v-else>You are not logged in</h3>
  </div>
</template>
<script>
export default {
  name: 'Greet',
  props: ["email"],
};
</script>
<style>
* {
  margin-top: 30px;
  text-align: center;
  font-size: 20px;
  font-family: 'Courier New', Courier, monospace;
}
</style>
```